spec/features/projects/viewing_project_spec.rb

```diff
@@ -45,6 +45,61 @@
     end
   end
 
+  context 'temporal label' do
+    context 'when a date range has been selected' do
+      before :all do
+        Capybara.reset_sessions!
+        load_page :search, project: ['C14758250-LPDAAC_ECS', 'C1000000000-LANCEAMSR2']
+        login
+        click_link "Temporal"
+        fill_in "Start", with: "2010-01-01 00:00:00\t\t"
+        fill_in "End", with: "2014-02-01 00:00:00\t\t"
+        wait_for_xhr
+        click_button('Apply Filter')
+        sleep 1
+        click_link 'My Project'
+      end
+      it 'shows the start and end dates of that range within the temporal label' do
+        expect(find('#temporal-label')).to have_content('Jan 01, 2010 - Feb 01, 2014')
+      end
+    end
+
+    context 'when a start date has been selected without an end date' do
+      before :all do
+        Capybara.reset_sessions!
+        load_page :search, project: ['C14758250-LPDAAC_ECS', 'C1000000000-LANCEAMSR2']
+        login
+        click_link "Temporal"
+        fill_in "Start", with: "2010-01-01 00:00:00\t\t"
```

Comment:
Are the tab characters necessary?

Reply:
Might not be - I can try it without however that particular code was a copy and paste from similar tests. Usage of tabs when setting temporal ranges is in multiple tests.